libera/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'groovy'
+apply plugin: 'application'
+apply plugin: 'maven'
+apply plugin: 'build-dashboard'
```

Comment:
we should move the `maven` plugin and `build-dashboard` plugin to the root build script